docs/source/parameters.rst

```diff
@@ -15,3 +15,80 @@ See `this release's parameter defaults <https://github.com/E3SM-Project/zppy/blo
 on GitHub for a complete list of parameters and their default values. 
 You can also view the most up-to-date, 
 `unreleased parameter defaults <https://github.com/E3SM-Project/zppy/blob/main/zppy/templates/default.ini>`_.
+
+Deprecated parameters
+=====================
+
+The following parameters no longer perform any function:
+   ::
+
+        "e3sm_to_cmip_environment_commands"
+        "ts_fmt"
+        "scratch"
+        "atmosphere_only"
+        "plot_names"
+
+Parameter checking & inferring -- for users
+===========================================
+
+There are two types of inferences, each with their own parameter in ``default.ini``:
+
+* ``infer_path_parameters``: infer paths that are not explicitly provided in the configuraiton file. Default is ``True``.
+* ``infer_section_parameters``: infer subtask dependency names that are not explicitly provided in the configuration file. Default is ``True``.
+
+**Section inferences**
+
+For the ``climo``, ``ts``, ``e3sm_to_cmip``, and ``e3sm_diags`` tasks:
+
+* If ``subsection`` (the name of the subtask) is undefined, just use the value of ``grid``.
+
+For the ``e3sm_to_cmip`` task:
+
+* If ``ts_subsection`` (the name of the ``ts`` subtask that this ``e3sm_to_cmip`` subtask is dependent on) is undefined, assume it had the same name as this ``e3sm_to_cmip`` subtask.
+
+For the ``ilamb`` task:
+
+* If ``ts_land_subsection`` (the name of the ``ts`` land-specific subtask that this ``ilamb`` task is dependent on), assume it is ``land_monthly``.
+* If ``e3sm_to_cmip_land_subsection`` (the name of the ``e3sm_to_cmip`` land-specific subtask that this ``ilamb`` task is dependent on), again assume it is ``land_monthly``.
+* If we are not doing a ``land_only`` run and ``ts_atm_subsection`` (the name of the ``ts`` atm-specific subtask that this ``ilamb`` task is dependent on), assume it is ``atm_monthly_180x360_aave``.
+* If we are not doing a ``land_only`` run and ``e3sm_to_cmip_atm_subsection`` (the name of the ``e3sm_to_cmip`` atm-specific subtask that this ``ilamb`` task is dependent on), again assume it is ``atm_monthly_180x360_aave``.
+
+
+**Path inferences**
+
+For the ``e3sm_diags`` task:
+
+* If ``reference_data_path`` (the path to the reference data) is undefined, assume it is the ``diagnostics_base_path`` from Mache plus ``/observations/Atm/climatology/``. (So, it is important to change this for model-vs-model runs).
+
+
+For the ``ilamb`` task:
+
+* If ``ilamb_obs`` (the path to observation data for ``ilamb``) is undefined, assume it is the ``diagnostics_base_path`` from Mache plus ``/ilamb_data``.
+
+**Required parameters, by e3sm_diags set**
+
+TODO: Insert main table from Confluence
+
+These parameters are required for all model-vs-model runs: ``diff_title``, ``ref_name``, ``short_ref_name``. 
+
+TODO: Insert model-vs-model table from Confluence
```

Comment:
https://acme-climate.atlassian.net/wiki/spaces/IPD/pages/4984209586/zppy+parameters+for+e3sm_diags

It's really difficult to create tables in `rst`, so for a big table, Confluence seemed like the better fit. For actually including it in the docs, I suppose we could add in screenshots?